package.json

```diff
@@ -1,37 +1,36 @@
 {
-  "author": "Allan Ebdrup",
-  "name": "nodeerrors",
+  "author": "Debitoor",
+  "name": "@debitoor/nodeerrors",
   "description": "error handling module for node",
-  "version": "2.1.3",
+  "version": "3.0.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/Muscula/nodeerrors"
+    "url": "https://github.com/debitoor/nodeerrors"
   },
   "private": false,
   "main": "./lib/nodeerrors.js",
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=16.20.0"
   },
   "dependencies": {
-    "moduleconfig": "^2.0.0",
-    "underscore": "^1.8.3",
-    "uuid": "^3.0.1"
+    "@debitoor/moduleconfig": "github:debitoor/moduleconfig#pass-start-dir-to-function",
```

Comment:
This will be updated to specific version after releasing https://github.com/debitoor/moduleconfig/pull/2